vars/createTemplateConfig.groovy

```diff
@@ -0,0 +1,17 @@
+def call(params, file_path = 'vm.vars') {
+
+  def vm_networks = params.vm_networks
+  def vm_hardware_memory_mb = params.vm_hardware_memory_mb
+  def vm_hardware_num_cpus = params.vm_hardware_num_cpus
+  def vm_hdd_size = params.vm_hdd_size
```

Comment:
Can you check if when `vm_hdd_size` does not exist in `params` map, it will throw an exception?